CHANGELOG.md

```diff
@@ -8,18 +8,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Add mapping between similar instance types `m4.16xlarge` and `m5.16xlarge`
+### Added
+
+- Add mapping between similar instance types `m4.16xlarge` and `m5.16xlarge`.
+- Add `lifecycle` label to the `aws_operator_ec2_instance_status` metric to distinguish on-demand and spot.
```

Comment:
Are these incidental changes from upgrading k8scloudconfig?

Reply:
I just moved this one into the correct section - the changelog was not created by me.

Reply:
Ah, missed that.